atc/integration/integration_suite_test.go

```diff
@@ -98,10 +104,11 @@ var _ = AfterEach(func() {
 	Expect(err).NotTo(HaveOccurred())
 
 	postgresRunner.DropTestDB()
+})
 
+var _ = AfterSuite(func() {
 	dbProcess.Signal(os.Interrupt)
-	err = <-dbProcess.Wait()
-	Expect(err).NotTo(HaveOccurred())
+	<-dbProcess.Wait()
```

Comment:
Is it a concern if the DB process can't be ended properly? I can see if the test is ran in a container in the pipeline it should be ok. How about running the tests locally? 

It seems some other suits are doing it this way
https://github.com/concourse/concourse/blob/3dc0c519c47f6f8017c81ea7ce97520e3b27c9f8/atc/db/migration/migration_suite_test.go#L39-L41

should them be the same?

Reply:
I went ahead and added some helpers to the `postgresrunner` package for common Ginkgo setup/teardown logic (and used the teardown with a timeout, so now all suites running postgres will timeout after 10 seconds). Lemme know what you think